rust/rust.grammar/src/org/netbeans/modules/rust/grammar/RustLanguageLexer.java

```diff
@@ -41,35 +40,55 @@ public class RustLanguageLexer extends AbstractAntlrLexerBridge<RustLexer, RustT
 
     private static final Logger LOG = Logger.getLogger(RustLanguageLexer.class.getName());
 
-    private static String formatMessage(String kind, Recognizer<?, ?> recognizer, Object o, int line, int charPositionInLine, String message, RecognitionException ex) {
-        return String.format("%s @%3d:%-3d %s", kind, line, charPositionInLine, message);
-    }
-
+    /**
+     * A BaseErrorListener that listens for errors in the lexer. We avoid
+     * throuing exceptions, since AbstractAntlrLexerBridge doesn't know how to
+     * handle them and this may interfere with the EDT. We enable logginf of
+     * errors instead, for debugging and enhancement purposes.
+     */
     private static final class RustLanguageLexerErrorListener extends BaseErrorListener {
 
+        private static final Level LEVEL = Level.FINE;
```

Comment:
Is `Level.FINE` intentional here, or leftover from your debugging sessions?

Reply:
D'oh! It's a leftover from my debugging sessions and because of doing commits being tired :-D. Will push in a minute. Thanks for the review!